### ISSUE TITLE
Error "could not connect to main window" occurred in popup.html using WebView

### ISSUE THREAD

**Issue by st-sunrize** [solid/node-solid-server#1193]
I would like to develop an solid app using Android or iOS app's WebView.
However, An error "could not connect to main window" occurred at popup.html when I try to log in to my pod in solid.community or my private v5.0.1 node-solid-server. 
The error does not occur when I log in by chrome or safari.
Could you give some advices to login to my pod using WebView ?

**Comment by jaxoncreed** [comment]
Sorry, right now we aren't supporting web views. But that's something we need to do in the near future.

**Comment by RubenVerborgh** [comment]
I wonder if this could help: https://github.com/solid/data-kitchen/commit/ec1668bf67e9b9c630e0a9237b05862396e47d08

**Comment by jaxoncreed** [comment]
@RubenVerborgh Yeah I'd say it's related, but I would anticipate that fixing one will not automatically fix the other.
